src/Formatter/AbstractFormatter.php

```diff
@@ -48,33 +34,4 @@ public static function accepts()
      * @return string
      */
     abstract protected function body(PayloadInterface $payload);
-
-    /**
-     * Get the response status from the payload.
-     *
-     * @param PayloadInterface $payload
-     *
-     * @return integer
-     */
-    public function status(PayloadInterface $payload)
```

Comment:
Actually I'm [totally wrong][1], these methods are always used publicly. 😨 I will update the PR.

[1]: https://github.com/equip/framework/blob/542d1c3f38f8871c18a00bd8d9161b0cb2009bf2/src/Responder/FormattedResponder.php#L131-L135